Add public goods funding page (#7732)
ethereum/ethereum-org-website: src/content/public-goods-funding/index.md

---
title: Public goods funding
description: An overview of public goods funding solutions on Ethereum
lang: en
template: use-cases
image: ../../assets/enterprise-eth.png
emoji: 🌎
sidebar: true
sidebarDepth: 2

> summaryPoint1: Public goods are nonexcludable and nonrivalrous and benefit everyone

Reviewer: ```suggestion
summaryPoint1: Public goods are non-excludable, non-rivalrous and benefit everyone
```

summaryPoint2: Traditional models of funding public goods are inefficient and encourage coordination failures
summaryPoint3: Ethereum provides useful infrastructure for improving funding and development of critical public goods
---

Public goods are commodities and services provided to all members of a society (without the possibility of direct profit) usually by government or private organizations. While the creation of public goods is critical to societal progress and wellbeing, traditional models of coordinating to fund public goods suffer problems that limit their efficiency and effectiveness. This makes it difficult to build useful public goods and results in worse outcomes that affect everyone.

> But there might be a solution to the problems of funding public goods: blockchain technology. By applying cryptoeconomic incentives and mechanism designs, new blockchain-based projects are fixing the broken model of traditional public goods funding and solving coordination failures that plague modern markets.

Reviewer: ```suggestion
But there might be a solution to the problems of funding public goods: blockchain technology. By applying cryptoeconomic incentives and mechanism designs, new blockchain-based projects are fixing the broken model of traditional public goods funding and solving coordination failures in modern markets.
```

## What is a public good? {#what-is-a-public-good}

As mentioned, public goods are defined as commodities or services made freely available to all members of a society. For a commodity or service to be classified as a public good, it must possess the folllowing characteristics:

> 1. **Nonexcludable**: A public good is _nonexcludable_ in the sense that it is costly or impossible to exclude any individual from accessing it. A public road is one example of a public good—no one can prevent you from driving on it (provided you have a license).

Reviewer: ```suggestion
1. **Non-excludable**: A public good is _non-excludable_ in the sense that it is costly or impossible to exclude any individual from accessing it. A public road is one example of a public good, no one can prevent you from driving on it (provided you have a license).
```

> 2. **Nonrivalrous**: A public good is _nonrivalrous_, meaning that one person consuming the commodity doesn't prevent another person from doing the same. Going back to the previous example: if you drive on a public road, that doesn't prevent another person from driving on the same road.

Reviewer: ```suggestion
2. **Non-rivalrous**: A public good is _non-rivalrous_, meaning that one person consuming the goods or service doesn't prevent another person from doing the same. Going back to the previous example: if you drive on a public road, that doesn't prevent another person from driving on the same road.
```

A **private good** is the opposite: they are _rivalrous_ and _excludable_. For example, if Alice eats a slice of pizza, then Bob cannot eat the same slice of (Alice and Bob are "rivals" in consumption). More so, by eating the slice of pizza, Alice prevents (or "excludes") Bob from eating it, making it excludable.

> Many public goods are also marked by their "positive externalities". An externality is an indirect cost or benefit borne by A positive externan meaning that i benefits of providing it outweigh the disadvantages.Other examples of public goods include:

Reviewer: ```suggestion
Many public goods are also marked by their "positive externalities". An externality is an indirect cost or benefit incurred or received by a third party. Positive externalities occur when the positive gain is received by both the social level and the private level. Examples of public goods include:
```

- Education
- Access to information
- Open-source software
- Roads
- Public healthcare
- Access to clean air

> - Law enforcement and national defense

Reviewer: ```suggestion

```

Somewhat debatable, might remove this

## Why is funding public goods important? {#importance-of-funding-public-goods}

Unlike [private goods](https://www.investopedia.com/terms/p/private-good.asp), public goods provide benefits for the majority, not just one individual. Such goods are necessary in creating a future that results in positive outcomes for all members of society. Public roads, for example, facilitate commerce and increase societal prosperity and wellbeing.

> Traditionally, public goods are usually made available by governments (through taxes collected from citizens). Private markets can also create public goods: the radio is an example of a public good—broadcast stations cannot prevent listeners from receiving signals (i.e., it is nonexcludable) and multiple individuals can listen at the same time (i.e., it is nonrivalrous).

Reviewer: ```suggestion
Traditionally, public goods are usually made available by governments (through taxes collected from citizens). Private markets can also create public goods: the radio is an example of a public good. Broadcast stations cannot prevent listeners from receiving signals (i.e., it is non-excludable) and multiple individuals can listen at the same time (i.e., it is non-rivalrous).
```

> However, the conventional models of building and funding public goods run into various problems. Not only have these issues stymied the development of public goods, but it has also resulted in public goods with [negative externalities](https://corporatefinanceinstitute.com/resources/knowledge/economics/negative-externalities/).

Reviewer: ```suggestion
However, the conventional models of building and funding public goods run into various problems. Not only have these issues discouraged the development of public goods, but it has also resulted in public goods with [negative externalities](https://corporatefinanceinstitute.com/resources/knowledge/economics/negative-externalities/).
```

> ### Problems of traditional public goods funding {problems-of-traditional-public-goods-funding}

Reviewer: ```suggestion
## Problems of traditional public goods funding {problems-of-traditional-public-goods-funding}
```

### 1. Misaligned incentives {#misaligned-incentives}

Traditional methods of funding public goods suffer from misaligned incentives among different entities coordinating to allocate resources. These incentive problems can be described using several concepts from economic theory:

#### The free-rider problem {#free-rider-problem}

> The [free-rider problem](https://bit.ly/3ATE3KC) describes a situation whereby some individuals refuse to pay towards building a public good, but intend to use it. In this case, such individuals are "taking a free ride" on the contributions of others towards funding the public good.

Reviewer: ```suggestion
The [free-rider problem](https://bit.ly/3ATE3KC) describes a situation where some individuals refuse to pay towards building a public good, but intend to use it. In this case, such individuals are "taking a free ride" on the contributions of others toward funding the public good.
```

The free-rider problem is central to the problem of funding public goods. If everyone refuses to pay for a public good, hoping someone else does, it becomes impossible to fund and build the public good.

#### Tragedy of commons {#tragedy-of-commons}

The [tragedy of commons](https://www.investopedia.com/terms/t/tragedy-of-the-commons.asp) concerns the degradation of commonly shared resources through overconsumption. A common resource is similar to a public good (they are nonexcludable), but can typically diminish in quality and availability as more people use it (i.e., they are rivalrous).

> Because individuals cannot prevent others from consuming the common resource, they are incentivized to take from the shared resource pool without replenishing it. But, as more people consume the resource, either of two things can happen: a) supply of the resource dwindles until there isn't any left for public consumption b) the quality of the resource degrades over time until it becomes unusable. Both situations are non-ideal, which is why solving the tragedy of commons is a key goal of new designs for funding public goods.

Reviewer: ```suggestion
Because individuals cannot prevent others from consuming the common resource, they are incentivized to take from the shared resource pool without replenishing it. But, as more people consume the resource, either of two things can happen: a) the supply of the resource reduces until there isn't any left for public consumption b) the quality of the resource degrades over time until it becomes unusable. Both situations aren't ideal, which is why solving the tragedy of commons is a key goal of new designs for funding public goods.
```

#### Democratic voting {#democratic-voting}

Funding models that apply democratic voting ("one person, one vote") rules rely on the majority opinion to allocate funding for public goods. In theory, using the collective opinion to guide funding is a great idea. In reality, however, this system has problems:

1. **Rational indifference**: In a democratic voting scheme, the ability of each individual to influence the outcome is limited (you can only vote once). As such, those whom a public good benefits the most (for example, public healthcare for aged citizens) are unable to push for its production. This leads to individuals rationalizing nonparticipation ("Why bother voting when my vote matters little?").

2. **Voter ignorance**: Prioritizing the preferences of the majority may not necessarily lead to spending on the right public goods owing to widespread voter ignorance. In this case, informed voters are unable to push for the production of more beneficial public goods.

> 3. **Negative externalities**: Another downside with democratic funding schemes is the cost borne by the minority. In democratic systems, those who may not necessarily need a public good are forced to pay the same as those who need it the most. For example, Bob (a risk-averse individual) is forced to pay taxes to subsidize medical procedures for Alice (a risk-seeking individual), even though Bob himself might never need such services.

Reviewer: I don't like this example. Public health is a great public good, and we all use it even if it's later in life.

Author: I just replaced with a less controversial example. Let me know if it works.

#### Plutocratic voting {#plutocratic-voting}

> Some funding models use a plutocratic mechanism in which an individual's financial power determines their influence on the process of providing public goods. These systems are typically described as "one dollar, one vote". In theory, plutocratic funding is ideal since it allows for expressing unequal preference for public goods—but it also has problems:

Reviewer: Is it ideal even in theory? I would very much argue it isnt.

Author: Like other forms of funding, I think this one has its own advantages (if it had too many drawbacks without any plausible advantage, no one would use it). Traditional fund-raising platforms (e.g., Kickstarter) are all examples of plutocratic voting (i.e., "putting your money where your mouth is"). And while they may favor those with wealth, they can be a good way for private markets to fund public goods (at least, that's what VC funding looks like in practice).

3. Every donation towards a project is matched with a corresponding amount of funding from the official matching pool. Instead of being matched 1:1, funds are matched according to the [quadratic funding formula](https://wtfisqf.com/?grant=10&grant=5,5&grant=2,2,2,2,2&grant=1,1,1,1,1,1,1,1,1,1&match=1000).

The formula works by adding the square roots of each contribution, squaring the result, and subtracting the original amount donated from it. The final figure represents the matched funding, i.e., how much money each project receives from the matching pool. We can explain this concept with an example:

Imagine two parks are planned for development, with the teams building each park (we call them `park A` and `park B`) requesting for funding. Ten people agree to donate $25 each to build `park A`, while two affluent investors donate $180 each to `park B`.

The table below shows how much each park will receive under a quadratic funding scheme:

> ![A diagram showing showing funds received during a quadratic funding round](./quadratic-funding-table.png)

Reviewer 2: I'm not sure I see the benefit of a table as an image when we could render this as text and translate it.
I would suggest using a table here:
```suggestion
| | No. of donors | Amount contributed by individual donors | Total contribution | Matched funding | Total funding received |
| :--- | :--- | :--- | :--- | :--- | :--- |
| Park A | 10 | $25 | $250 | $2,250 | $2,500 |
| Park B | 2 | $400 | $800 | $800 | $1,600 |
```

Reviewer: +1. Let me know if you want me to do this @emmanuel-awosika.

Author: Looks good to me.

##### Accounting for differences in spending power and degree of preferences

Naive implementations of democracy (one-person-one vote) and plutocracy (one-dollar-one-vote) fail to account for differences in wealth and degree of preference. Quadratic funding fixes this problem to an extent as seen in the example described earlier:

i. Those who wanted `park A` donated what they could afford without losing the power to properly express their preference for that public good. Quadratic funding empowers _every_ individual (irrespective of wealth) to influence the allocation of capital to public goods. This solves the issue of rational indifference and increases willingness to participate in collective action projects.

ii. Other donors who wanted `park B` were able to express the strength of their preferences for that public good by contributing more money to its development. Perhaps, unknown to most people, `park B` has more long-term benefits for the community than `park A`. In this scenario, quadratic voting/funding prevents widespread ignorance from canceling out the wisdom of informed citizens.

> #### Reducing negative externalities

Reviewer 2: If I recall we don't need these past H4, so the above ones should be fine
```suggestion
#### Reducing negative externalities {#reducing-negative-externalities}
```

But what if it was possible to publish results of an on-chain vote, while concealing specifics of individual votes—without reducing the integrity of the process? This is precisely what [Minimum Anti-Collusion Infrastructure](https://ethresear.ch/t/minimal-anti-collusion-infrastructure/5413) seeks to achieve (albeit with some trust assumptions).

Using MACI, it is possible to tally results from a quadratic voting round without needing to publish a breakdown of votes cast by each user. This is achieved using a combination of Ethereum smart contracts for recording and aggregating votes from users and [zero-knowledge proof](/zero-knowledge-proofs/) circuits that prove the vote tallying was performed correctly.

MACI also allows users to submit multiple votes with the caveat that only votes encrypted with a valid keypair will be processed. Hence, a bribee could vote for a briber's choice using an invalid keypair, but the latter cannot know if the vote was valid or not (since the final tally doesn't say how each voter voted). Here's how the [MACI documentation](https://github.com/privacy-scaling-explorations/maci/tree/master/specs) describes it:

> > When Alice casts her vote, she signs her vote with her private key, encrypts her signature with Dave's public key, and submits the result to the smart contract.

Reviewer 2: Not a problem with this PR, but noting that we don't actually style these blockquotes.

<img width="1567" alt="image" src="https://user-images.githubusercontent.com/54227730/215612514-b690111c-8387-45de-a684-5245ba79bbb0.png">

The `> ` marker does wrap these in `<blockquote>` tags, but we're not styling them... @nloureiro Do we have something like this in the design system?

> This is how GitHub styles these, as an example

Reviewer 3: Not yet. Something to be added very soon, for sure

- [GreenPilled: How Crypto Can Regenerate The World](https://www.amazon.com/GreenPilled-How-Crypto-Regenerate-World/dp/1034928163) — _Book on regenerative cryptoeconomics by Kevin Owocki_

### Podcasts and videos {#podcasts-and-videos}

- [Reflections on the State of Public Goods Funding](https://www.youtube.com/watch?v=H05VKLiaXd8) — _Kevin Owocki & Juan Benet’s talk at HackFS 2022 Summit_
- [Funding Public Goods: Algorithms and Mechanisms](https://www.youtube.com/watch?v=zxzDIXv_8jk) — _Vitalik Buterin’s speech at [Funding The Commons](https://fundingthecommons.io/), a virtual summit for individuals and organizations building new models of sustainable public goods funding and value alignment in open source networks._
- **[Green Pill](https://availableon.com/greenpill)** — _Podcast exploring the intersection of programmable money, game theory, and mechanism design in the search for new ways to fund, design, develop, & market regenerative web3-era applications and digital assets._

### Communities {#communities}

- [Gitcoin on Discord](https://discord.com/invite/gitcoin)

> - [Clr.fund on Discord](https://discord.gg/ZnsYPV6dCvhttps://discord.gg/ZnsYPV6dCv)

Reviewer 2: ```suggestion
- [Clr.fund on Discord](https://discord.gg/ZnsYPV6dCv)
```
